bitbots_hcm/bitbots_hcm/hcm_dsd/hcm_blackboard.py

```diff
@@ -4,142 +4,112 @@
 
 import numpy
 from ament_index_python import get_package_share_directory
-from bitbots_hcm.fall_checker import FallChecker
-from bitbots_hcm.fall_classifier import FallClassifier
-from diagnostic_msgs.msg import DiagnosticArray, DiagnosticStatus
 from geometry_msgs.msg import PointStamped, Twist
 from rclpy.action import ActionClient
 from rclpy.node import Node
+from rclpy.task import Future
 from rclpy.time import Time
 from sensor_msgs.msg import Imu, JointState
 from std_msgs.msg import Empty as EmptyMsg
 from std_srvs.srv import Empty as EmptySrv
 from std_srvs.srv import SetBool
 
-from bitbots_msgs.action import Dynup
-from bitbots_msgs.msg import JointCommand
-from humanoid_league_msgs.action import PlayAnimation
-from humanoid_league_msgs.msg import RobotControlState
+from bitbots_msgs.action import Dynup, PlayAnimation
+from bitbots_msgs.msg import Audio, RobotControlState
 
 
 class HcmBlackboard():
     def __init__(self, node: Node):
         self.node = node
+
+        # Basic state
         self.current_state: RobotControlState = RobotControlState.STARTUP
         self.stopped: bool = False
-        self.shut_down_request: bool  = False
-        self.simulation_active = self.node.get_parameter('simulation_active').value
-        self.visualization_active = self.node.get_parameter('visualization_active').value
-
-        # this is used to prevent calling Time a lot, which takes some time
-        # we assume that the time does not change during one update cycle
-        self.current_time = self.node.get_clock().now()
-        self.start_time = self.current_time
-        # Imu
+
+        # Save start time
+        self.start_time: Time = self.node.get_clock().now()
+
+        # Get parameters
+        self.simulation_active: bool = self.node.get_parameter('simulation_active').value
+        self.visualization_active: bool = self.node.get_parameter('visualization_active').value
+        self.pickup_accel_threshold: float = 1000.0  # TODO: make this a parameter
```

Comment:
This is already a param, at least it exists in the config file

Reply:
Done